src/main/webapp/js/form.js

```diff
@@ -0,0 +1,70 @@
+(function ($) {

```

Comment:
do we actually use anything in this file? I can't seem to see where else we reference the css classes.

Reply:
yes, I believe this is required for the form to load. some third-party library calls this.